parameter_files/WebParameters_2dii.yml

```diff
@@ -1,9 +1,9 @@
 default:
     paths:
-        project_location_ext: /Users/jacobkastl/Documents/git/PACTA_analysis/
-        data_location_ext: /Users/jacobkastl/Documents/git/pacta-data/2019Q4/
-        template_location: /Users/jacobkastl/Documents/git/create_interactive_report/
-        stress_test_location: /Users/jacobkastl/Documents/git/StressTestingModelDev/
+        project_location_ext: /home/mauro/git/PACTA_analysis/
+        data_location_ext: /home/mauro/git/pacta-data/2019Q4/
+        template_location: /home/mauro/git/create_interactive_report/
+        stress_test_location: /home/mauro/git/StressTestingModelDev/
```

Comment:
there's got to be a better way than repeatedly switching between user specific paths

Reply:
I agree.

(BTW, What you see here happens automatically, but it shouldn't be here -- I forgot to add this file to my list of `--assume-uncahged` files.)

Reply:
I managed to work around this problem by using relative paths on my local clone, for instance...
```
        project_location_ext: ../PACTA_analysis/
        data_location_ext: ../pacta-data/2019Q4/
        template_location: ../create_interactive_report/
        stress_test_location: ../StressTestingModelDev/
```

Reply:
That's a great idea.